src/styles/utils/loops.scss

```diff
@@ -0,0 +1,19 @@
+@each $day, $firstDay in $days {
+  .calendar--start-day--#{$day}
+  .calendar__day:first-child {
+    margin-left: #{($cell-size + $calendar-gap) * $firstDay};
+  }
+}
+
+@for $index from 1 through 31 {
+  .calendar__day:nth-child(#{$index})::before {
+    content: "#{$index}";
+  }
+}
+
+@for $index from 28 through 31 {
```

Comment:
This loop dosn't work. U should use space before `:nth-child`. And u can also use `display: none`